app/main/routes.py

```diff
@@ -15,6 +17,9 @@
 
 from app.main import bp
 from app.main.forms import CookiesForm
+
+from app.data.data import consignment_response, consignment_files_response
```

Comment:
This will be removed, was a temporary data store until opensearch integration was completed.